be/src/common/config.cpp

```diff
@@ -1160,13 +1160,13 @@ DEFINE_mDouble(high_disk_avail_level_diff_usages, "0.15");
 DEFINE_Int32(partition_disk_index_lru_size, "10000");
 // limit the storage space that query spill files can use
 DEFINE_String(spill_storage_root_path, "${DORIS_HOME}/storage");
-DEFINE_mInt64(spill_storage_limit, "10737418240"); // 10G
-DEFINE_mInt32(spill_gc_interval_ms, "2000");       // 2s
+DEFINE_mInt32(spill_storage_usage_percent, "20"); // 20%
```

Comment:
Do not write config like this.
Maybe there are 2 requirements:
1. Set the usage by percent like you do.
2. Set the usage by size, such as 10g, 10T, 100mb

If you add a config like this, then you has to add another config spill_storage_usage_bytes when meet req 2.
If user set both of them, we do not know which should be used.

You could refer the config mem_limit.